### ISSUE TITLE
KASPA not shown after fresh installation of OneKey wallet on windows 7

### ISSUE THREAD

**Issue by jk-85** [OneKeyHQ/app-monorepo#4088]
After I import my 12-words passphrase into a brand new 4.19.0-2024011597 OneKey wallet on windows 7 and click on "BNB"-chain, I immediately see my tokens there. But: When I do that for the KASPA-chain, the wallet shows me a green button with a "+"-symbol to create a new KASPA.wallet. After clicking on that "+"-symbol, the old KASPA-wallet is shown correctly. But the user wants to display their funds immediately, has to be the same behaviour as clicking on the BNB-chain. Nevertheless, great wallet ;-)

**Comment by ByteZhang1024** [comment]
No plan for support at the moment.


BNB belongs to the EVM account. EVM is the default account created. 
Kaspa needs to create an additional account by yourself.

**Comment by jk-85** [comment]
You understand it maybe wrong. Would be better if I had created a screenshot as it happens. After I imported my account and clicked on that KASPA network the green "+" button was shown. If I'm forced to click on that button it should be better labeled as "See your KASPA funds" or something, because after that I see my funds and was worried why the funds are not displayed immediately.
![grafik](https://github.com/OneKeyHQ/app-monorepo/assets/13134932/60ce2dfd-d626-42f4-ae40-1d02a2eca976)

Have a nice day! Keep up the work!
